### PR TITLE
suit-2022.5.2-release

### PR DESCRIPTION
**Pull request by robyww**
2022.5.2 release

### CODE REVIEW

File: config/app.config [hunk 1]
```diff
@@ -3,7 +3,7 @@
 //---------------------------------------------
 BuildMajor = 2022
 BuildMinor = 5
-BuildRev = 1
+BuildRev = 2
```
Comment (gpdf) [inline config/app.config:6]: Please update; we already have 2022.5.2 on data-int.

File: config/app.config [hunk 2]
```diff
@@ -65,7 +65,7 @@ ehcache.peerDiscovery = "PubSub"
 redis.host = "redis-srv"
 ehcache.replicate = true
 // point a IRSA for testing - I will be overriddien in a real deployment
-lsst.hips.masterUrl="https://irsa.ipac.caltech.edu/data/hips/list"
+lsst.hips.masterUrl="https://data-int.lsst.cloud/api/hips/list"
```
Comment (gpdf) [inline config/app.config:68]: I would use data.lsst.cloud here and then, when needed, override it in the Kubernetes config for data-int.

File: src/suit/js/SUIT.js [hunk 3]
```diff
@@ -157,12 +124,16 @@ let options = {
...
-        defHipsSources: {source: 'lsst', label: 'Featured'},
+        defHipsSources: {source: 'lsst', label: 'Rubin Featured'},
+        mergedListPriority: 'lsst',
         adhocMocSource: {
-            sources: getDefaultMOCList(),
-            label: 'Featured MOC '
+            sources: [
+                'temp://lsst/dp02_dc2/hips/images/color_gri',
+                'temp://lsst/dp02_dc2/hips/images/band_u',
+                'temp://lsst/dp02_dc2/hips/images/band_g',
```
Comment (gpdf) [inline src/suit/js/SUIT.js:133]: This should be the whole list of 7 (gri,u,g,r,i,z,y).  At a later date we should figure out how to make this more metadata-configurable.

File: src/suit/js/SUIT.js [hunk 4]
```diff
@@ -81,46 +80,14 @@ AND ( t_min <= 61000 AND 60900 <= t_max )
...
-    'GAVO', 'HSA', 'NED', 'NASA Exoplanet Archive']);
+const tapServices=  [
+    lsstEntry,
+    ...getTAPServices( ['IRSA', 'Gaia', 'CADC', 'MAST Images', 'GAVO', 'HSA', 'NED', 'NASA Exoplanet Archive'])
+];
 
 
-const TAP_PATH= 'api/tap';
-
-/**
```
Comment (gpdf) [inline src/suit/js/SUIT.js:92]: What's going on with the removal of all this logic?

File: src/suit/js/SUIT.js [hunk 5]
```diff
@@ -33,12 +33,11 @@ var props = {
...
 props = mergeObjectOnly(props, window.firefly?.app ?? {});
-const {template}= props;
 
-const lsstEntry= (label, url) => (
+const lsstEntry=
     {
-        label,
-        value: url,
+        label:'LSST RSP',
+        value: 'https://data-int.lsst.cloud/api/tap',
```
Comment (gpdf) [inline src/suit/js/SUIT.js:40]: Worried about the hard-coding here

File: src/suit/html/suit-test.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
+<!doctype html>
```
Comment (gpdf) [inline src/suit/html/suit-test.html:1]: This looks like a really good idea for testing!

**Review by gpdf — CHANGES_REQUESTED** [review]

File: src/suit/js/SUIT.js (hunk 5 above)
Comment (robyww) [reply src/suit/js/SUIT.js:40]: This should always be overridden in deployment. Therefore the values are just the default.

File: src/suit/js/SUIT.js (hunk 4 above)
Comment (robyww) [reply src/suit/js/SUIT.js:92]: It is really just a hack and is it is not longer needed with out better deployment methods.

File: src/suit/js/SUIT.js (hunk 3 above)
Comment (robyww) [reply src/suit/js/SUIT.js:133]: I use a sub list to better indicate when the whole list was not picked up from the portal/templates/deployment. We should never be relying on this list. It should always be overridden.

File: config/app.config (hunk 2 above)
Comment (robyww) [reply config/app.config:68]: ok, but it should be always overridden.

File: config/app.config (hunk 1 above)
Comment (robyww) [reply config/app.config:6]: ok

File: config/app.config (hunk 2 above)
Comment (gpdf) [reply config/app.config:68]: In the actual config files, there's a template variable for "instance root URL" used, so in practice there's no need for pretty much ever typing "data-int" directly in the config files.

Comment (gpdf) [reply config/app.config:68]: - name: "PROPS_lsst__hips__masterUrl"
   value: "{{ .Values.global.baseUrl }}/api/hips/list"

https://github.com/lsst-sqre/phalanx/blob/13da8890c38bc3f3289e55102fda6649b85edfa9/services/portal/templates/deployment.yaml#L49

File: src/suit/js/SUIT.js (hunk 3 above)
Comment (gpdf) [reply src/suit/js/SUIT.js:133]: That makes sense; so I'll take a look at what in phalanx for this.

File: config/app.config (hunk 2 above)
Comment (gpdf) [reply config/app.config:68]: See below - I get what you're doing now.

**Pull request merged**